### ISSUE TITLE
Drop the no_edge_features assembly leg: it is the 1024d graph without edge_attr

### ISSUE THREAD

**Issue by jeff1evesque** [jeff1evesque/pyg-knowledge-graph-builder#411]
## Problem

**Every run builds the same graph twice.** After the seed,
`notebook/multi_experiment.ipynb` submits two `--mode pyg_only` jobs from the same
enriched Parquet: `no_edge_features` (`edge_feature_config.enabled: false`) and
`baseline_1024d`. Both use the same node feature settings. The first is the second
without its `edge_attr` tensors.

Compared on run 20260910T214553Z, from the files the run published
(`checksums.json`, `graph_schema.json`, the node index, and a SHA-256 of every tensor
stored in both `.pt` files):

- same 9,447,814 nodes, 70,805,631 edges and 837 edge types, with the same count for
  every edge type; the node index matches row for row
- `normalization.json`, `ontology_schema.json`, `slot_mapping.json` and
  `encoding_config.json` are byte-identical, so both graphs carry the same encoding
  digest
- 991 of the 992 tensors the two files share are byte-identical. The 1024d file's
  only extra content is 63 `edge_attr` tensors: 18,102,764 edges × 32 × 4 =
  2,317,153,792 bytes (2.16 GiB). That is the whole size difference between the
  files, 42,148,716,724 against 39,831,544,839 bytes, to within 18 KB
- the one tensor that is not byte-identical, `market_quotes_OptionSnapshot.x`
  (9,317,030 × 1024), differs in 30 of its 9,540,638,720 values, each by the
  smallest float32 step (at most 7.5e-09). Both jobs collected the same
  1,395,042,347 feature entries. That is rounding, not different data (see Out of
  scope)

**What the second copy costs.**

- time: the job took 41.4 min on run 20260905T130719Z and 43.7 min on run
  20260910T214553Z
- storage: 39.96 of the 94.91 GB published for run 20260910T214553Z, about 42% (the
  `.pt` and metadata 39.83 GB, the node index 0.13 GB)

**The reason it was added no longer holds.** The comment above `EXPERIMENTS` in cell
`cell-10` runs it first as a memory probe: "if the cheap one cannot fit, the
expensive ones certainly cannot, and finding that out first costs 12 minutes instead
of an hour." Today:

- it takes 41–44 minutes, not 12
- the 1024d job builds node features before edge features
  (`spark_jobs/pyg_builder/constructor.py:263` and `:322`), so any failure the probe
  can hit, the 1024d job hits at the same step
- the probe cannot find a failure caused by edge features, since it does not build
  them
- the other expensive leg, `all_edge_categories`, was dropped on 2026-08-29, so one
  job is left to protect

## What already exists

**A graph without edge features is the 1024d graph with `edge_attr` ignored.** A
model that does not pass `edge_attr` to its layers already ignores it. To drop the
tensors from a loaded graph:

```python
import torch

data = torch.load("1024d/hetero_data_1024d.pt", map_location="cpu",
                  weights_only=False, mmap=True)
for edge_type in data.edge_types:
    if "edge_attr" in data[edge_type]:
        del data[edge_type].edge_attr
```

`mmap=True` maps the file instead of reading it, so tensors that are never used are
never read into memory.

**`edge_feature_config.enabled: false` still works**
(`spark_jobs/pyg_builder/edge_feature_extractor.py:686`), for a build that should
skip the edge-feature step.

**Nothing downstream expects two variants.**

- `bin/publish_run.py:150` finds variants by filename, so a run with one `.pt`
  publishes one folder
- `bin/record_run_outcome.sh:118` writes `all N submissions ok` for any N, and
  `bin/publish_run.py:132` accepts any N
- the 1024d job reads nothing the dropped job wrote: each variant writes its own
  files, and each job gets its own checkpoint subdirectory
  (`spark_jobs/build_graph.py:780` sets the parent; Spark adds one per application)
- `tests/test_notebook_seed_flags.py:175` reads cell `cell-10` by its id;
  `tests/test_publish_run.py` uses the name only as sample data

## Proposal

Remove `no_edge_features` from `EXPERIMENTS`, so a run is the seed and one assembly
job, and say in the docs that the graph without edge features comes from ignoring
`edge_attr`.

No build code changes, so no cluster run is needed to check the change; CI covers
it. The next run is the first with one assembly job, and its outcome and published
size are the measurement.

## Code changes

`notebook/multi_experiment.ipynb`, cell `cell-10` only. Keep the cell id, and commit
the notebook with no outputs.

1. Delete the `no_edge_features` entry from `EXPERIMENTS`.
2. Replace the "ORDERED CHEAPEST FIRST" paragraph with a dated note, in the style of
   the `all_edge_categories` note below it. For example:

   ```python
   # One assembly leg. no_edge_features was dropped on YYYY-MM-DD (#<this issue>). It
   # built this same graph without edge_attr: on run 20260910T214553Z every node, edge
   # and node index row matched, 991 of 992 shared tensors were byte-identical, and the
   # 63 edge_attr tensors (2.16 GiB) were the only difference, for 43.7 minutes and
   # 39.96 GB of the published run. It ran first as a memory probe, but this leg builds
   # node features before edge features, so it hits any failure the probe could find
   # at the same step. For a graph without edge features, ignore edge_attr.
   ```

3. In the `PYG_SEED_ONLY` comment below the list, "costs one leg instead of three"
   becomes "instead of two".

`feature-410` also edits `cell-10`, adding an `EXCLUDE_NODE_TYPES` block just after
the list. If git reports a conflict in that cell when the second branch merges, keep
both changes.

## Unit tests

None. `tests/test_notebook_seed_flags.py` already reads `cell-10` and checks it still
calls `flag_set("PYG_SEED_ONLY")`.

## Documentation

**`documentation/reference/outputs.md`, Published Runs.** Remove the two
`no_edge_features/` lines from the tree (`:98-99`), and add after the tree:

````markdown
**One graph per run, with edge features.** There is no separate graph without edge
features: it is this one with `edge_attr` ignored. A model that does not pass
`edge_attr` to its layers already ignores it. To drop the tensors from a loaded graph:

```python
import torch

data = torch.load("1024d/hetero_data_1024d.pt", map_location="cpu",
                  weights_only=False, mmap=True)
for edge_type in data.edge_types:
    if "edge_attr" in data[edge_type]:
        del data[edge_type].edge_attr
```

The node features, the node index and the encoding digest match what a build with
`edge_feature_config.enabled: false` writes. Check the `.pt` against
[`checksums.json`](#checksumsjson) before loading it: `torch.load` with
`weights_only=False` runs whatever the pickle contains.
````

**`documentation/operations/running-a-job.md`, "Example — edge features disabled"
(`:139`).** Add after the example:

```markdown
The default build already holds this graph: it is the same graph with `edge_attr`
ignored (see [Published Runs](../reference/outputs.md#published-runs)). Turn edge
features off only to skip the edge-feature step when nothing will read them.
```

## Acceptance criteria

- [ ] `EXPERIMENTS` in `cell-10` holds only `baseline_1024d`; the cell id is unchanged
      and the notebook is committed with no outputs
- [ ] the next run's `outcome.txt` reads `all 2 submissions ok`, and its published
      folder holds `1024d/` and no `no_edge_features/`
- [ ] that run's wall time and published size are recorded here, against run
      20260910T214553Z: 170.7 min from launch to notebook exit (seed 65.9,
      `no_edge_features` 43.7, `baseline_1024d` 59.2) and 94.91 GB published
- [ ] mkdocs pages updated, `mkdocs build --strict` clean
- [ ] the fast suite passes in CI

## Out of scope

- **Removing `edge_feature_config.enabled`.** The flag stays; only the leg goes.
- **Published runs already written.** They keep their `no_edge_features/` folder.
  The identity that publishes cannot delete (#407).
- **Metadata from a build with edge features off contradicts itself.**
  `graph_schema.json`'s summary counts 63 edge types with features, and
  `feature_spec.json` lists the same 63 under `edge_types_with_features` beside
  `enabled: false`, while every edge entry says `has_features: false`. The count
  comes from the edge types allowed to have features, not the ones built
  (`spark_jobs/pyg_builder/metadata_writer.py:452`). Its own issue.
- **Node features are not bit-for-bit repeatable at full size.** The 30 one-step
  differences above contradict `documentation/reference/outputs.md:192`, which says
  feature tensors "compared equal". At about 3 values in a billion, the e2e twin
  runs on small fixtures would not show it. Harmless for trai
